docs/core/testing/unit-testing-mstest-runner-intro.md

```diff
@@ -0,0 +1,238 @@
+
+    <!-- Replace coverlet collector 
+      <PackageReference Include="coverlet.collector" Version="6.0.0" />
+    --> 
+    <PackageReference Include="Microsoft.Testing.Platform.Extensions.CodeCoverage" Version="17.9.4-beta.23563.1" />
+  </ItemGroup>
+
+</Project>
+```
+
+Building the application will generate an executable application for your test project. For example `Contoso.MyTests.exe` on Windows.
+
+## Running & Debugging tests
+
+> [!IMPORTANT]
+> Telemetry is by default collected by the runner, see [Telemetry](./unit-testing-mstest-runner-telemetry.md) to learn more.
+
+MSTest Runner test projects are built as executables that can be run (or debugged) directly. There is no additional test running console or command that is needed.
+
+The application will exit with non-zero exit code in case of an error. For a list of known exit codes see [exit codes](unit-testing-mstest-runner-exit-codes.md).
+
+This is how you run tests in common scenarios:
+
+### In console
+
+#### Running the application directly in Console
+
+Publishing the test project using `dotnet publish` and running the application directly is another way to run your tests. For example `./Contoso.MyTests.exe`. In some scenarios it is also viable to use `dotnet build`, to produce the executable, but there can be edge cases to that for example when using Native AOT.
+
+#### Using `dotnet run`
+
+`dotnet run` can be used to build and run your test project. This is the easiest, although sometimes slow, way to run your tests. Using `dotnet run` is practical when you are editing and running tests locally because it ensures that the test project is re-built when needed. `dotnet run` will also automatically find the project in the current folder.
+
+#### Using `dotnet exec`
+
+`dotnet exec` can be used to run an already built test project, this is an alternative to running the application directly. `dotnet exec` requires path to the built test project dll.
+
+> [!IMPORTANT]
+> Providing path to the test project executable (.exe) will result in an error:
+
+> ```plaintext
+> Error:
+>   An assembly specified in the application dependencies manifest
+>   (Contoso.MyTests.deps.json) has already been found but with a different
+>   file extension:
+>     package: 'Contoso.MyTests', version: '1.0.0'
+>     path: 'Contoso.MyTests.dll'
+>     previously found assembly: 'S:\t\Contoso.MyTests\bin\Debug\net8.0\Contoso.MyTests.exe'
+> ```
+
+#### Using `dotnet test`
+
+> > [!IMPORTANT]
+> `dotnet test` is currently working only with MSTest, NUnit and xUnit tests.
+
+For tests authored with MSTest, NUnit or xUnit test framework, it is possible to run tests with `dotnet test`.
+
+Provide a path to the tested dll, or to the project and your tests will run.
+
+### In Visual Studio
+
+Testing.Platform tests can be run (and debugged) in VisualStudio, they integrate with Test Explorer, and can also be run directly as Startup Project.
```

Comment:
I am not sure how to best word that. The idea is to differentiate MSTest tests produced the old way to test produced as self-contained test app.

Besides, MSTest runner is based on Microsoft.Testing.Platform solution that we hope to offer as alternative to VSTest also for xUnit and NUnit but we are waiting to get enough feedback on MSTest side before opening it further.

Reply:
I've added a suggestion of a quick note about Microsoft.Testing.Platform to the first paragraph. But it feels like there should be more explanations about what it is. 

I am also not sure how to properly let the user know that this is not an article about vstest.

Reply:
Maybe adding another article that describes vstest vs testing.platform, and call out the namespaces used, and fundamental differences?

Reply:
Pushed the basics of what I think should be covered in a new file in this PR. But it is hard for me to judge what to put there , because I am so close to the project.